src/vivarium_gates_nutrition_optimization/constants/data_values.py

```diff
@@ -1,5 +1,15 @@
+from typing import NamedTuple
+
+
 ############################
 # Disease Model Parameters #
 ############################
+class _Durations(NamedTuple):
+    ## Days
+    FULL_TERM = 40 * 7
+    POSTPARTUM_DURATION = 6 * 7
+    DETECTION = 6 * 7
+    PARTIAL_TERM = 24 * 7
+
 
-POSTPARTUM_DURATION = 6 * 7  ## Days
+DURATIONS = _Durations()
```

Comment:
Are any of these going to be used long-term besides post-partum?

Reply:
I think PARTIAL_TERM, FULL_TERM, and DETECTION will remain used in a similar context to what they're being used for now, to set the scaling factors for the pregnancy duration draws, even as the full term distribution gets more complicated with e.g. LBWSG